app/models/course/registration.rb

```diff
@@ -0,0 +1,61 @@
+class Course::Registration < ActiveRecord::Base
+  stampable
+
+  scope :staff, -> { where(role: [roles[:teaching_assistant], roles[:manager], roles[:owner]]) }
```

Comment:
you might want to make the `Roles` type in `CourseUser` return you an array of roles that match the given characteristics.

Also, maybe we might want to reason out the categories of users. What do you propose?